src/unix/linux_like/linux/mod.rs

```diff
@@ -1195,6 +1195,19 @@ s! {
         pub rsv: [::c_uint; 5],
     }
 
+    pub struct ptp_clock_caps {
```

Comment:
Nit, trying to do a bit more tagging of where everything comes from (for the consts too)

```rust
    // ptp_clock.h
    pub struct ptp_clock_caps {
```

Reply:
it's on line 1181 already, I think that should do? I've assumed that if the comment is missing it's from the same file as the preceding definition?

Reply:
Oh I missed that, thanks that's good 👍